api/tests/routes/native/v1/account_test.py

```diff
@@ -698,7 +715,9 @@ def test_cannot_confirm_after_expiration(self, client, app):
     def test_cannot_confirm_if_no_pending_email_update(self, client, app):
         user = users_factories.BeneficiaryGrant18Factory()
         expiration_date = datetime.utcnow() + users_constants.EMAIL_CHANGE_TOKEN_LIFE_TIME
-        token = encode_jwt_payload({"current_email": user.email, "new_email": "exemple@exemple.com"}, expiration_date)
+        token = encode_jwt_payload(
+            {"current_email": user.email, "new_email": "exemple@exemple.com"}, expiration_date
+        )  # TODO also test with a token with a different format
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Missing TODO ?